backend/wmg/pipeline/constants.py

```diff
@@ -119,3 +119,23 @@
     "UBERON:0000014",  # zone of skin
     "UBERON:0000916",  # abdomen
 ]
+
+
+ORGANISM_INFO = [
+    {"label": "homo_sapiens", "id": "NCBITaxon:9606"},
+    {"label": "mus_musculus", "id": "NCBITaxon:10090"},
+]
+
+
+class CensusParameters:
```

Comment:
What was your reasoning for making this a class?

Reply:
Easy to patch for testing.